.github/workflows/tests.yaml

```diff
@@ -87,7 +87,6 @@ jobs:
             - name: Run Tests
               timeout-minutes: 10
               run: |
-                  ifconfig -a
```

Comment:
Why this removal? Is it a problem?

Reply:
It's just noise and I've seen people trying to copy/paste it into other places, thinking it needed to be run in "run:" blocks....